src/Yasumi/Yasumi.php

```diff
@@ -120,7 +120,7 @@ public static function nextWorkingDay(string $class, DateTime $startDate, int $w
         $date = $startDate instanceof DateTime ? clone $startDate : new DateTime($startDate);
 
         // Default to 1
-        $workingDays = is_null($workingDays) ? 1 : $workingDays;
+        $workingDays = $workingDays ?? 1;
```

Comment:
The Null Coalescing Operator was introduced in PHP7. At the moment Yasumi still supports PHP5 so this will cause BC issues.

Reply:
Yes. At first I didn't use null coalescing, but than I thought it would look nicer and since all my projects depend on php7.1, I did't thought twice.